src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -298,31 +327,33 @@ describe("Offer InitialMutation", () => {
     })
 
     it("tracks viwing the low offer speedbump", async () => {
+      const trackData = {
+        action_type: "Viewed offer too low",
+        flow: "Make offer",
+        order_id: "1234",
+      }
       await page.setOfferAmount(1000)
 
-      expect(mockPostEvent).not.toHaveBeenCalled()
+      expect(mockPostEvent).not.toHaveBeenLastCalledWith(trackData)
```

Comment:
We have a few tracking events firing now (see tests for `PriceOptions` component), so we have to be more specific when checking.